articles/client-platforms/jquery/_includes/_new_api.html

```diff
@@ -0,0 +1,5 @@
+<% if (account.userName) { %>
+  <p>In the APIs section in <a href="${manage_url}/#/">dashboard</a>, click the Create API button. Provide a Name and Identifier for your API. You must choose the RS256 signing algorithm. Once it is created, navigate to the Scopes tab and create the applicable Scopes for your API.</p>
```

Comment:
Can we put these in bold:
* Create API
* Name
* Identifier
* Scopes

"create the applicable Scopes" => "create the applicable scopes"